tests/playwright/src/model/ai-lab-page.ts

```diff
@@ -0,0 +1,139 @@
+    this.openAIAppButton = this.applicationDetailsPanel.getByRole('button', { name: 'Open AI App' });
+    this.deleteAIAppButton = this.applicationDetailsPanel.getByRole('button', { name: 'Delete AI App' });
+  }
+
+  async waitForPageLoad(): Promise<void> {
+    await playExpect(this.heading).toBeVisible();
+    await playExpect(this.navigationBar).toBeVisible();
+    await playExpect(this.recipesCatalogButton).toBeVisible();
+    await playExpect(this.runningAppsButton).toBeVisible();
+    await playExpect(this.catalogButton).toBeVisible();
+    await playExpect(this.servicesButton).toBeVisible();
+    await playExpect(this.playgroundsButton).toBeVisible();
+  }
+
+  async openRecipesCatalog(): Promise<void> {
+    await playExpect(this.recipesCatalogButton).toBeVisible();
+    await this.recipesCatalogButton.click();
+    await this.page.waitForTimeout(2000);
+    await playExpect(this.recipesCatalogPage).toBeVisible();
+    await playExpect(this.recipesCatalogContent).toBeVisible();
+    await playExpect(this.recipesCatalogNaturalLanguageProcessing).toBeVisible();
+    await playExpect(this.recipesCatalogAudio).toBeVisible();
+    await playExpect(this.recipesCatalogComputerVision).toBeVisible();
+  }
+
+  async openRecipesCatalogApp(category: Locator, appName: string): Promise<void> {
+    await this.openRecipesCatalog();
+    await category.getByRole('region', { name: 'content', exact: true }).first().getByRole('region', { name: appName, exact: true }).getByRole('button', { name: 'More details' }).click({ timeout: 5_000 });
+    await playExpect(this.webview.getByRole('heading', { name: appName })).toBeVisible();
+  }
+
+  /**
+   * @throws {Error} If the app details are not visible or Delete AI App button is not visible
+   */
+  async tryDeleteOpenApp(): Promise<void> {
+    await playExpect(this.applicationDetailsPanel).toBeVisible();
+    console.log('Application details panel is visible, pressing delete button.');
+    await this.deleteAIAppButton.click({ timeout: 5_000 });
+    try {
+      console.log('Waiting for Delete app confirmation dialog to appear.');
+      await playExpect(this.page.getByRole('dialog', { name: 'Podman AI Lab' })).toBeVisible();
+      console.log('Delete app confirmation dialog is visible, pressing Confirm button.');
+      await this.page.getByRole('dialog', { name: 'Podman AI Lab' }).getByRole('button', { name: 'Confirm' }).click({ timeout: 5_000 });
+    } catch(error) {
+      console.warn(`Warning: Delete app confirmation dialog interaction failed, possibly app is not installed.\n\t${error}`);
+    }
+    console.log('Waiting for AI App Removed message to appear.');
+    await playExpect(this.applicationDetailsPanel).toContainText('AI App Removed', { timeout: 30_000 });
+    console.log('Checking Start AI App button visibility.');
+    await playExpect(this.startRecipeButton).toBeVisible();
+  }
+
+  async startOpenApp(): Promise<void> {
+    await this.startRecipeButton.click({ timeout: 5_000 });
```

Comment:
I think default timeout is 5 sec, so no need to explicitly state that?

Reply:
some of the cases were only timing out after 30 seconds with no explicit timeout

Reply:
Unless the default timeout is changed that really shouldnt be happening.

Reply:
```suggestion
    await playExpect(this.startRecipeButton).toBeEnabled();
    await this.startRecipeButton.click();
```

Reply:
thank you for the suggestion @cbr7 updated in new page fragment